core/plugins/org.polarsys.capella.core.ui.properties.richtext/plugin.xml

```diff
@@ -82,4 +82,80 @@
             class="org.polarsys.capella.core.ui.properties.richtext.CapellaMDERichtextWidgetEditorImpl">
       </editorWidgetContribution>
    </extension>
+   <extension
+         point="org.eclipse.ui.commands">
+      <command
+            categoryId="org.polarsys.capella.core.platform.sirius.ui.navigator"
+            defaultHandler="org.polarsys.capella.core.ui.properties.richtext.handlers.CopyAsDescriptionLinkHandler"
+            id="org.polarsys.capella.core.ui.properties.richtext.CopyAsDescriptionLink"
+            name="%command.copyAsDescriptionLink.name ">
```

Comment:
Is it working with a space at the end?

Reply:
Yes it works, but I will remove the space.